src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -153,10 +163,17 @@ class ManageApplicationContainer extends React.Component<
   }
 
   public render() {
-    const { mode, hackerDetails, submitted, pageNumber, loaded } = this.state;
+    const {
+      mode,
+      hackerDetails,
+      submitted,
+      pageNumber,
+      loaded,
+      settings,
+    } = this.state;
     return loaded ? (
       // If application creation deadline of Jan 3, 2020 11:59:59PM EST has passed or form is submitted, return user to the home page
```

Comment:
nit: remove this line, outdated and could be misleading

Reply:
Done